cpu/cc2538/clock.c

```diff
@@ -74,8 +86,7 @@ static volatile uint64_t rt_ticks_startup = 0, rt_ticks_epoch = 0;
  *
  * We also initialise GPT0:Timer A, which is used by clock_delay_usec().
  * We use 16-bit range (individual), count-down, one-shot, no interrupts.
- * The system clock is at 16MHz giving us 62.5 nano sec ticks for Timer A.
- * Prescaled by 16 gives us a very convenient 1 tick per usec
+ * The prescaler is computed according to the system clock.
```

Comment:
```
 * The prescaler is computed according to the system clock in order to get 1
 * tick per usec.
```

There is another reference to `16 MHz` in the `cc2538-clock` group description right after the file header.